cegs_portal/search/templates/search/v1/reg_effect.html

```diff
@@ -106,9 +105,8 @@ <h1>
 
     {% if regulatory_effect.targets.exists %}
     <div class="container">
-        <div class="flex max-w-full min-w-full justify-between items-center">
-            <div class="justify-self-start"></div>
-            <div class="table-title flex items-center justify-center">
+        <div class="table-title" style="display: flex; align-items: center; justify-content: space-between; position: relative;">
+            <div style="display: flex; align-items: center; justify-content: center; flex-grow: 1;">
```

Comment:
Please don't use inline style tags. Either use the relevant tailwind classes or, if you find you're using the same group of styles over and over, put those styles in a new CSS class and use that class.